modules/ugdk-core/src/graphic/rendertarget.cc

```diff
@@ -32,13 +34,13 @@ void RenderTarget::Clear(structure::Color color) {
 }
 
 void RenderTarget::Bind() {
-    system::AssertCondition<system::InvalidOperation>(!IsActive(), "RenderTarget must be inactive for Bind.");
+    //system::AssertCondition<system::InvalidOperation>(!IsActive(), "RenderTarget must be inactive for Bind.");
```

Comment:
Are you sure it's ok to remove this assertion?

Reply:
Yes, nowhere else does the code check for the render target being active